### ISSUE TITLE
Change "Lexicographical Order" (and related) to "Unicode code point order".

### ISSUE THREAD

**Issue by gkellogg** [w3c/json-ld-api#552]
This was raised against another spec in https://github.com/w3c/rdf-canon/issues/18, and a similar change should be made here and in Framing. Note that this shouldn't affect any actual test results.

The definition from rdf-canon is the following:

> **Unicode code point order**
> This refers to determining the order of two Unicode strings (A and B), using [Unicode Codepoint Collation](https://www.w3.org/TR/xpath-functions/#codepoint-collation), as defined in [[XPATH-FUNCTIONS](http://localhost:8000/spec/#bib-xpath-functions)], which defines a [total ordering](https://en.wikipedia.org/wiki/Total_order) of [strings](http://localhost:8000/spec/#dfn-string) comparing code points.

Also defines other terms related to "lexical".

**Comment by gkellogg** [comment]
Summary: Clarify that "lexicographical order" means "code point order" and change some wording to reflect.

See #574.

**Comment by anatoly-scherbakov** [comment]
Should this issue be closed since the PR is already merged?

**Comment by gkellogg** [comment]
Yes, I think I kept it open because I wasn't sure how the errata features would work with a closed issue.

**Comment by aphillips** [comment]
Please do not remove the I18N labels. It messes with the automation. I'm closing our matching issue today. No need to reopen this.
